src/main/java/robot/subsystems/Arm.java

```diff
@@ -167,4 +167,15 @@ public void enableRampRate() {
         shoulderMotor.setClosedLoopRampRate(RAMP_RATE);
     }
 
+    public double getTemperatures(int index) {
```

Comment:
Add javadocs to all the getters and possibly rename to something like getMotorTemperature that is slightly less ambiguous.